packages/@aws-cdk/aws-codebuild/test/test.project.ts

```diff
@@ -1021,7 +1021,10 @@ export = {
 
       'can be provided as a verbatim full secret ARN followed by a JSON key'(test: Test) {
         // GIVEN
-        const stack = new cdk.Stack();
+        const app = new cdk.App();
+        const stack = new cdk.Stack(app, 'ProjectStack', {
+          env: { account: '123456789012' },
+        });
```

Comment:
Can we leave the existing tests as they are?

I'm fine with adding an assertion to them, but _not_ with changing any pre-conditions they have.

Same comment below.

Reply:
Yeah we won't need this with the change to suggested below 👍🏿